packages/components/src/segmented-control/test/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { SegmentedControl } from '../index';
+
+describe( 'props', () => {
+	const options = [
+		{ value: 'olaf', label: 'O' },
+		{ value: 'samantha', label: 'S' },
+	];
+
+	test( 'should render correctly', () => {
+		let container;
+
+		act( () => {
```

Comment:
Is wrapping the rendering of the component inside `act()` necessary?

Reply:
It seems not. I'll update. Thanks!